### DISCUSSION TITLE
Type error when setting up Sentry

### DISCUSSION THREAD

**Discussion by Nick-Mazuk** [vercel/next.js#17975] [Help] [answered]
I'm following along to the [with sentry](https://github.com/vercel/next.js/tree/canary/examples/with-sentry) example and getting a TypeScript type error with `_app.tsx`:

```tsx
export default function App({ Component, pageProps, err }: AppProps): JSX.Element => {
    // Workaround for https://github.com/vercel/next.js/issues/8592
    return <Component {...pageProps} err={err} />
}
```

```
Property 'err' does not exist on type 'AppPropsType<Router, {}>'
```

Is this an actual type error, or is there a bug in the `AppProps` type definition? There does not appear to be a runtime error.

**Comment by JavierMartinz** [comment, marked as answer]
You can extend `AppProps`, e.g:

`type Props = AppProps & { error: never } `

**Reply by Nick-Mazuk** [reply]
That does seem to be working, but it does feel hacky to be modifying Next.js's types.
